src/infrastructure/services/Cache.ts

```diff
@@ -0,0 +1,13 @@
+export class Cache {
+  public setItem(key: string, value: string) {
+    localStorage.setItem(key, value);
+  }
+
+  public getItem(key: string) {
+    return localStorage.getItem(key);
+  }
+
+  public clearItem(key: string) {
```

Comment:
Why not naming this `removeItem` as the native one?

Reply:
It doesn't really matter it's a generic interface for the application browser storage. What's important is to not rely directly on localStorage as we might want to change implementations in some cases (like private navigation on Safari does not allow localStorage, someday we might implement the cache using indexDB or whatnot)